pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -107,6 +107,8 @@ type getconfigContext struct {
 	radioSilence     types.RadioSilence // the intended state of radio devices
 	triggerRadioPOST chan Notify
 
+	localInfoPOSTTicker flextimer.FlexTickerHandle
```

Comment:
Why do we need to do this periodically as opposed to only when some app status changes?

Reply:
Well, I think we can hit the situation when local profile app is not ready or not updated to use info endpoint. In this case we will send all info periodically assuming that it will eventually be received.

Reply:
That seems very suboptimal. 
If the user changes the local-profile-server config (or disables then re-enables it), will that not result in sending all of the application status?
That seems preferably than having a periodic task.

Reply:
Well, this is the only one service per EVE and I hope having POST here once per hour is not the problem from perspective of load. But from perspective of some kind of keep-alive (and intermediate solution before all local servers will be updated) it is nice to have.

Reply:
OK